### PR TITLE
Fixing bug 1282 Save then Load from database (PostgreSQL) adds extra "\" to existing text

### PR DESCRIPTION
**Pull request by mariprudencio**
Changed "DBExporter.java" to don't add extra "\" in postgreSQL

### CODE REVIEW

**Comment by simonharrer** [comment]
Thank you for your pull request. 

Can you please add yourself to the list of contributors as shown in CONTRIBUTING.md

And the last thing: can you please add an item to the CHANGELOG describing your bug fix in short?

File: src/main/java/net/sf/jabref/sql/exporter/DBExporter.java [hunk 1]
```diff
@@ -133,17 +134,21 @@ private void populateEntriesTable(int database_id,
...
+						val = val.replace("\'", "''");
+						val = val.replace("`", "\\`");
+					}
 					query = query + "'" + val + "'";
 				} else {
 					query = query + "NULL";
 				}
 			}
 			query = query + ", '" + database_id + "');";
+			System.out.println("\nServer Type: " + dbStrings.getServerType());
```
Comment (stefan-kolb) [inline src/main/java/net/sf/jabref/sql/exporter/DBExporter.java]: This should be logged rather than send to standard output.

**Comment by stefan-kolb** [comment]
https://sourceforge.net/p/jabref/bugs/1282/

**Comment by igorsteinmacher** [comment]
Moved to PR #198

**Pull request closed without merging**